src/ImageSharp/Formats/Gif/GifMetadata.cs

```diff
@@ -45,9 +50,15 @@ private GifMetadata(GifMetadata other)
     public GifColorTableMode ColorTableMode { get; set; }
 
     /// <summary>
-    /// Gets or sets the length of the global color table if present.
+    /// Gets or sets the global color table, if any.
+    /// </summary>
+    public ReadOnlyMemory<Color>? GlobalColorTable { get; set; }
```

Comment:
Is color resolution different from 24 bits ever expected according to the gif standard? If not, wouldn't it be better to go with `ReadOnlyMemory<Rgb24>` and avoiding the conversions?

Reply:
It doesn't and I did consider using `Rgb24` originally but thought that `Color` gave us greater portability on the API surface. For example, someone could now easily create a pallet quantizer from this color table.

Reply:
If we think such use-cases will be common it might be justified, but then I would extend the xml doc describing that `GlobalColorTable` is 24bit. For me it still feels better to expose the raw gif data as-is, and instead provide helper API-s to make work with `PaletteQuantizer` easier.